deluge/ui/gtkui/torrentdetails.py

```diff
@@ -90,13 +89,13 @@ def __init__(self):
         ]
 
         self.translate_tabs = {
-            "All": _("_All"),
-            "Status": _("_Status"),
-            "Details": _("_Details"),
-            "Files": _("_Files"),
-            "Peers": _("_Peers"),
-            "Options": _("_Options"),
-            "Trackers": _("_Trackers")
+            "All": _("All"),
```

Comment:
honestly i just thought it looked better in the gui, why have _ in menu items? is there a conflict somewhere?